apps/webpack.js

```diff
@@ -20,7 +21,9 @@ var baseConfig = {
     "blockly": "this Blockly",
     "react": "var React",
     "react-dom": "var ReactDOM",
-    "jquery": "var $"
+    "jquery": "var $",
+    "radium": "var Radium",
+    "bindings": true
```

Comment:
Shouldn't some of these stop being externals, now that code-studio is merged?  The whole point of this was that those libraries (like React and Radium) were provided by the code-studio bundle; can we now remove them here and let Webpack do the rest?